profile/README.md

```diff
@@ -1,8 +1,304 @@
+- **Timelines**: Stay up-to-date with our [Release Planning](https://github.com/orgs/eclipse-tractusx/projects/26) and [Release Schedule](https://github.com/orgs/eclipse-tractusx/projects/26/views/35).
+
+## Our Community and Governance
+
+Eclipse Tractus-X operates under the open governance model of the [Eclipse Foundation](https://www.eclipse.org/), ensuring transparency, fairness, and inclusivity in decision-making.
+
+- **[Code of Conduct](../CODE_OF_CONDUCT.md)**: We strive to maintain a welcoming and inclusive environment for everyone.
+- **Committers and Contributors**: Interested in becoming a committer? Check out the [Eclipse governance model](https://www.eclipse.org/membership/become-a-member/committer/) and [learn](https://www.eclipse.org/projects/training/) how you can play a key role in the development of our projects.
+- **Security**: Please report any security issues to **TBD** following our [TRG 8 - Security](https://eclipse-tractusx.github.io/docs/release/trg-8/trg-8-01) and [sig-security](https://github.com/eclipse-tractusx/sig-security).
+
+## Useful Links
+
+- [Tractus-X Official Website](https://www.tractus-x.com)
+- [Getting Started](https://eclipse-tractusx.github.io/docs/getting-started)
+- [Eclipse Foundation](https://www.eclipse.org)
+- [Eclipse Foundation Project handbook](https://www.eclipse.org/projects/handbook/)
+- [Catena-X Automotive Network e.V.](https://catena-x.net)
+- [Catena-X - Library](https://catenax-ev.github.io/)
+- [Eclipse Tractus-x Project Page](https://projects.eclipse.org/projects/automotive.tractusx)
+
+## Stay Connected
+
+- Join our mailing list: [subscribe here](https://accounts.eclipse.org/mailing-list/tractusx-dev)
+- Follow us on our [webpage](https://eclipse-tractusx.github.io/) and our [news sectcion](https://eclipse-tractusx.github.io/blog)
+- Join our matrix chats at: [Tractus-X](https://matrix.to/#/#automotive.tractusx:matrix.eclipse.org)
+
+## Our Community
+
+Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) and our project leads of Eclipse Tractus-X:
+
+| ![Stephan Bauer](https://images.weserv.nl/?url=avatars.githubusercontent.com/stephanbcbauer?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Mathias Brunkow Moser](https://images.weserv.nl/?url=avatars.githubusercontent.com/matbmoser?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/evegufy?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Daniel Miehle](https://images.weserv.nl/?url=avatars.githubusercontent.com/danielmiehle?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Björn Roy](https://images.weserv.nl/?url=avatars.githubusercontent.com/giterrific?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d)  |
```

Comment:
I wouldn't single out the project leads here so much, as our community is driven by committers, contributors and project leads. In regards to community, I'd find links to our Matrix chat and dev mailing list more meaningful.

Reply:
@evegufy, @matbmoser  I updated the description and added the links to mailinglist and matrix chat :) -> The project lead faces are still there :) Do you really want to delete them? :)

Reply:
OK :) will delete us/them :)